native/cocos/bindings/jswrapper/v8/Object.cpp

```diff
@@ -100,7 +100,188 @@ class JSBPersistentHandleVisitor : public v8::PersistentHandleVisitor {
     }
 };
 
-Object::Object() { //NOLINT
+// MapOperation
+MapOperation::MapOperation(Object *obj) {
+    if (obj->isMap()) {
+        _v8Map = v8::Map::Cast(*obj->_getJSObject());
+    } else {
+        SE_LOGE("se::Object(%p) is not a Map object", obj);
+    }
+}
+
+void MapOperation::clear() {
+    if (_v8Map == nullptr) {
```

Comment:
Why need to check it in every function?

Reply:
To make sure all methods of `MapOperation` doesn't crash if passing a `nullptr` or  an `invalid Map se::Object` to construct it. For instance:
```c++
MapOperation op(nullptr); // passing nullptr directly, _v8Map is nullptr
op.clear(); // Crash
```

```c++
se::Object* plainObj = se::createPlainObject();
MapOperation op(plainOBj); // Not a map object, _v8Map is nullptr
op.clear(); // Crash
```

Reply:
> MapOperation op(nullptr);

Support passing nullptr? If so, then it will crash, as the constructor doesn't check the passed in value.

Reply:
Good catch, needs to add `nullptr` check in constructor.

Reply:
FIXED

Reply:
FIXED

Reply:
Ok. Can i know why need to support passing nullptr?